src/components/equipment.tsx

```diff
@@ -1,434 +1,498 @@
+function FusionImage({ fusionimage }: { fusionimage: Models.Equipment }) {
   const rarityColors = new Map<string, string>([
-    ["Common",     "#FFFFFF"],
-    ["Uncommon",   "#68D5ED"],
-    ["Rare",     "#B36BFF"],
-    ["Unique",     "#FF00FF"],
+    ["Common", "#FFFFFF"],
+    ["Uncommon", "#68D5ED"],
+    ["Rare", "#B36BFF"],
+    ["Unique", "#FF00FF"],
     ["Epic", "#FFB400"],
     ["Chronicle", "#FF6666"],
     ["Legendary", "#FF7800"],
     ["Mythic", "#FFB400"],
   ]);
   return (
-    <div style ={{
-    }}>
-      <Image style = {{
-        border: `1px outset ${rarityColors.get(baseitemimage.itemRarity)}`,
-      }}
-        src={"https://img-api.dfoneople.com/df/items/" + baseitemimage.itemId}
-        width={28}
-        height={28}
-        unoptimized={true}
-        alt="item image"
-      />
+    <div style={{}}>
+      {fusionimage.upgradeInfo?.itemId != undefined && (
+        <Image
+          style={{
+            border: `1px outset ${rarityColors.get(fusionimage.itemRarity)}`,
+          }}
+          src={
+            "https://img-api.dfoneople.com/df/items/" +
+            fusionimage.upgradeInfo?.itemId
+          }
+          width={28}
+          height={28}
+          unoptimized={true}
+          alt="item image"
+        />
+      )}
     </div>
-  )
+  );
 }
-function AllItemimage({itemimage}: {itemimage: Models.Equipment}) {
-  // returns both base and fusion equipment images
+// basic 105 gear options
+function Options({ option }: { option: Models.Option }) {
+  const damage = option.damage;
   return (
-    <div style = {{
-      display: 'flex',
-    }}>
-    <div className="equipmentImageContainer m-1 p-1" style = {{
-      display: 'flex',
-    }}>
+    <div className="border m-0.5 p-1">
+      <div>
+        <h1 style={{ color: "white" }}>{`Level: ${option.level}`}</h1>
+      </div>
       <div>
-      <BaseItemImage baseitemimage={itemimage}/>
+        {`Damage Value:`}
+        <span style={{ color: "blue" }}>{`${damage} `}</span>
+        {`Buff: ${option.buff}`}
+      </div>
+      <div>{`${option.explainDetail}`}</div>
+    </div>
+  );
+}
+//ispin fusion
+function Option3({ option3 }: { option3: Models.Option3 }) {
+  return (
+    <div>
+      <div>Ispins Fusion Effects:</div>
+      {`${option3.explainDetail}`}
     </div>
+  );
+}
+// bakal fusion
+function Option2({ option2 }: { option2: Models.Option2 }) {
+  return (
+    <div>
+      <div>Bakal Fusion Effects:</div>
+      {`${option2.explainDetail}`}
     </div>
-    <div className="equipmentImageContainer m-1 p-1" style = {{
-      display: 'flex',
-    }}>
+  );
+}
+// dimension fusion
+function Option4({ option4 }: { option4: Models.Option4 }) {
+  return (
     <div>
-      <FusionImage fusionimage={itemimage}/>
+      <div>Hall of Dimensions Fusion Effects:</div>
+      {`${option4.explainDetail}`}
     </div>
+  );
+}
+// bakal weapon fusion
+function Option5({ option5 }: { option5: Models.Option5 }) {
```

Comment:
These Options types are very helpful at a glance. May want to refactor in the future for our sakes.